test/test_unistd.rs

```diff
@@ -335,6 +336,21 @@ fn test_lseek64() {
     close(tmpfd).unwrap();
 }
 
+#[cfg(target_family = "unix")]
+#[test]
+fn test_acct() {
+    skip_if_not_root!("test_acct");
+    let file = NamedTempFile::new().unwrap();
+    let path = file.path().to_str().unwrap();
+
+    acct::enable(path).unwrap();
+    Command::new("echo").arg("Hello world");
+    acct::disable().unwrap();
+
+    let md = metadata(path).unwrap();
+    assert!(md.len() > 0);
```

Comment:
This is a race.  acct is an asynchronous interface.  There's no guarantee that the accounting file is updated by the time that `echo` returns.

Reply:
Would a sleep be appropriate here?

Reply:
No, you need a polling loop.

Reply:
Would [inotify](https://github.com/inotify-rs/inotify) be okay to use?

Reply:
Just for a simple test case?  No, that's much too heavyweight.

Reply:
I used a simple loop, hopefully that works...